### ISSUE TITLE
Dynamically assign READ weight to elected Primary when single writer and writer_is_also_reader =1 

### ISSUE THREAD

**Issue by Tusamarco** [Tusamarco/pxc_scheduler_handler#12]
In case we have writer_is_also_reader=1 and set the preferred primary node in the READ HG to the lowest weight, after failover we can see the former primary is getting the lowest number of reads and the new elected primary get the full read load. is there a way to make the actual primary node always the least read heavy one? from what I can understand we can't achieve that with a simple weight setting.

In short 
weight for read in HG 8xxx
node1 = 100
node2 = 1000
node3 = 1000

Node1 is primary.
Node1 fails (or goes into maintenance) 
Node2 become Primary but now Node2 has read weight = 1000
We need to change read weight to the lowest given it is now the Primary

**Comment by Tusamarco** [comment]
Analyzing

**Comment by Tusamarco** [comment]
when the option preservePrimaryValues is enabled [default] the values of the previous Primaries for weight, connections and so on will be adopted by the new fail-over node. 
When in multi primary, all the primaries will have the same values.

to change:
- configuration files
- configuration objects for cluster
- DataObject PxcClusterImplementation will contain a representation of the Primary node with the values 
- In evaluate Writers when new elected(s) is valid then apply previous primary values
Node will be inserted in write group with parameters as previous Primary

**Comment by Tusamarco** [comment]
Implemented and delivered https://github.com/Tusamarco/pxc_scheduler_handler/commit/8fca7ec9e558ca8da318f80eaa0a84a91184e460
